common/dynamicconfig/constants.go

```diff
@@ -2525,4 +2525,10 @@ WorkerActivitiesPerSecond, MaxConcurrentActivityTaskPollers.
 		false,
 		`When set to true, logs all RPC/request errors for the namespace, not just unexpected ones.`,
 	)
+
+	ActivityAPIsEnabled = NewGlobalBoolSetting(
```

Comment:
You might as well make it per-namespace

Reply:
This is "feature flag". I don't see any value at doing this "per namespace"

Reply:
Sure there is, we often release previews of features to specific namespaces

Reply:
will do